developer-docs-site/docs/concepts/basics-accounts.md

```diff
@@ -2,42 +2,107 @@
+## Creating an account
 
-## Initial Account Setup
+When a new account is requested by the user, the [Aptos SDK](https://aptos-labs.github.io/ts-sdk-doc/classes/AptosAccount.html) will perform a series of cryptographic steps to create the account. Below is a conceptual view of these steps:
 
-An Aptos account is referenced by an account address, which is a 32-byte value. The account address is derived from the initial public verification key(s) for that account. Specifically, the account address is the 32-byte of the SHA-3 256 cryptographic hash of the initial public verification key(s) concatenated with a signature scheme identifier byte. The Aptos Blockchain supports two signature schemes: [Ed25519](/reference/glossary#ed25519) and MultiEd25519 (for multi-signature transactions). The account's private key is necessary for signing transactions.
+- Start by generating a new private key, public key pair.
+- From the user get the preferred signature scheme for the account: If the account should use a single signature or if it should require multiple signatures for signing a transaction. 
+- Combine the public key with the user's signature scheme to generate a 32-byte authentication key. 
+- Initialize the sequence number to 0. The sequence number represents the next transaction sequence number, to prevent replay attacks of transactions. Both the authentication key and the sequence number are stored in the account as an initial account resource. 
+- Create the 32-byte account address from the initial authentication key. 
 
-Each account also stores a `sequence_number`, which represents the next transaction sequence number to prevent replay attacks of transactions. This is initialized to `0` at account creation time.
+From now on, the user should use the private key for signing the transactions with this account. 
 
-## Authentication Keys
+## Signature schemes
 
-Each account stores an authentication key. This authentication key enables account owners to rotate their private key(s) associated with the account without changing the address that hosts their account. During rotation, the authentication key is updated based upon the newly-generated private, public key-pair(s).
+An account can send transactions. The Aptos Blockchain supports two signature schemes: 
 
-### Single signer authentication
+1. The [Ed25519](https://ed25519.cr.yp.to/) for single signature transactions, and 
+2. The MultiEd25519, for multi-signature transactions. 
 
-To generate an authentication key and account address:
+:::note
+The Aptos Blockchain defaults to single signature transactions.
+:::
+
+  
+## Rotating the keys
+
+For an existing account you can rotate the private key, public key pair, i.e., use a new pair of private, public keys at regular intervals. To rotate the keys you must pass the current authentication key of your account. This will generate and store a new authentication key in the account. **However, the account address will remain unchanged.**
```

Comment:
This feels like a bit of a throwaway section, it's enough to sort of explain that you can do it but doesn't explain how or why. Also I think it should come later, since the previous section was talking about signature schemes.

Reply:
Added the Account creation section. We can add more stuff to these sections as the doc evolves organically.